hack/local-development/start-apiserver

```diff
@@ -24,15 +24,18 @@ tls_dir="$__tmp_tls_dir"
 trap cleanup_kubeconfig EXIT
 trap cleanup_tls_dir EXIT
 
+# TODO: Make cluster identity unique for every
```

Comment:
Left over TODO? This would be quite important for the local dev scenario.

Reply:
well actually I added the ${USER} in the end do you think it is enough?

Reply:
Is ${USER}  truly uniqueue?

Reply:
Certainly not, but how unique do we need it to be. I guess in a single Gardener instance it would be enough. Would you like to suggest something else to be used instead.

Reply:
I guess you are right, it only has to be unique within one gardener installation.